Userland/Games/2048/main.cpp

```diff
@@ -37,9 +39,8 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 
     Config::pledge_domains("2048");
 
-    TRY(Core::System::pledge("stdio rpath recvfd sendfd"));
```

Comment:
It is needed for launching the Help app.

I don't like it either and have been thinking if we can have some other service that an app could ask to open the help page instead of launching the Help app.

Reply:
Why not just adds `unix` to the list of pledges?

Reply:
Because this pledge only removes `unix` from the earlier pledge

Reply:
Sorry for the confusion, must have misread!